src/Components/Users/ManageUsers.tsx

```diff
@@ -831,20 +789,21 @@ function UserFacilities(props: { user: any }) {
               </div>
             </div>
           )}
-          {!user?.home_facility_object && facilities.length === 0 && (
-            <div className="my-2 flex h-96 flex-col content-center justify-center align-middle">
-              <div className="w-full">
-                <img
-                  src="/images/404.svg"
-                  alt="No linked facilities"
-                  className="mx-auto w-80"
-                />
+          {!user?.home_facility_object &&
+            (userFacilities ? userFacilities.length === 0 : 0) && (
```

Comment:
The logic looks complex

```suggestion
          {!user?.home_facility_object && !userFacilities?.length && (
```